### DISCUSSION TITLE
Podman Package not found in `mcr.microsoft.com/devcontainers/base:noble|jammy`

### DISCUSSION THREAD

**Discussion by mathieu-benoit** [containers/podman#24846] [Q&A]
When using a Devcontainer in a GitHub Codespace based on `mcr.microsoft.com/devcontainers/base:noble` or `mcr.microsoft.com/devcontainers/base:jammy`, if I run `sudo apt update && sudo apt -y install podman`, I'm getting this error:
```
E: Unable to locate package podman
```

Could you guide me to how I can install `podman` in there please?

Thanks in advance!

Context: I have a DevContainer/Codespace config to have Docker pre-configured, but I want to have the same for Podman.

_Note: I saw this other discussion similar to that https://github.com/containers/podman/discussions/22973, but now closed without any answer for resolution._

**Comment by mathieu-benoit** [comment]
Hi @afbjorklund and @lsm5, as I see your names in https://github.com/containers/podman/discussions/22973 and https://github.com/containers/podman/issues/21041, any idea or pointer for my issue please? Thanks!

**Reply by afbjorklund** [reply]
Those bugs were for 20.04, where Podman was not included in the system repositories. But it should be, for 22.04 and 24.04

**Reply by mathieu-benoit** [reply]
Interesting, your comment led me to create this minimal `Containerfile`:
```
FROM mcr.microsoft.com/devcontainers/base:noble
RUN sudo apt update && sudo apt -y install podman
```
And indeed I'm able to build this container without any issue... weird, I'll look closer on why I'm not able to install `podman` on this:
```
{
	"name": "Dev Container with Podman",
	"image": "mcr.microsoft.com/devcontainers/base:noble",
	"features": {
	},
	"remoteEnv": {
        "PODMAN_USERNS": "keep-id"
      },
    "containerUser": "vscode"
}
```

Out of curiosity, do you know if anyone has tried to get a Devcontainer with Podman and shared an example somewhere?

**Reply by afbjorklund** [reply]
Probably want to ask that from devcontainers instead?

**Reply by mathieu-benoit** [reply]
Yes, good point, closing this one though. Thanks anyway!
